Lib/glyphsLib/filters/eraseOpenCorners.py

```diff
@@ -53,25 +58,44 @@ def endPath(self):
                 ix = ix + 1
                 continue
 
+            logger.debug("Considering line segment (%i,%i)-(%i,%i)" % (*segs[ix][0], *segs[ix][1]))
```

Comment:
if you're going to keep the logging in, at least try not to use the normal `%` string interpolation which is done all the time, but let logging module do it only when the message is actually emitted.
that is, replace:

```diff
- logging.debug("%s %s" % ("hello", "world"))
+ logging.debug("%s %s", "hello", "world")

Reply:
Oh, that's nice.